### PR TITLE
added product page

### PR DESCRIPTION
**Pull request by farahh001**
added product page.still working on the structure

### CODE REVIEW

File: techeria_controller/templates/watch.html [hunk 1]
```diff
@@ -1,236 +1,509 @@
 {% load static %}
+
```
Comment (sanjidanisha) [inline techeria_controller/templates/watch.html:2]: Rename to product.html

File: techeria_controller/templates/watch.html [hunk 2]
```diff
@@ -1,236 +1,509 @@
...
+    <!-- Bootstrap -->
+    <link rel="stylesheet" href="/static/watch_css/css/bootstrap.min.css">
+
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="/static/watch_css/css/font-awesome.min.css">
+
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="/static/watch_css/css/owl.carousel.css">
+    <link rel="stylesheet" href="/static/watch_css/style.css">
+    <link rel="stylesheet" href="/static/watch_css/css/responsive.css">
```
Comment (sanjidanisha) [inline techeria_controller/templates/watch.html:33]: link to appropriate css file name

**Comment by farahh001** [comment]
[🐛 This Card is a Bug](https://trello.com/c/9ytCnqPL/5-%F0%9F%90%9B-this-card-is-a-bug)

**Comment by farahh001** [comment]
[🐛 This Card is a Bug](https://trello.com/c/9ytCnqPL/5-%F0%9F%90%9B-this-card-is-a-bug)

File: techeria_controller/templates/watch.html (hunk 1 above)
Comment (farahh001) [reply techeria_controller/templates/watch.html:2]: ok

File: techeria_controller/templates/watch.html (hunk 2 above)
Comment (farahh001) [reply techeria_controller/templates/watch.html:33]: I am not sure which one you meant.

Comment (farahh001) [reply techeria_controller/templates/watch.html:33]: I am not sure which one you meant

**Pull request merged**